Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -534,7 +534,7 @@ public void Update(EntityUid uid, PlantHolderComponent? component = null)
             foreach (var (gas, amount) in component.Seed.ExudeGasses)
             {
                 environment.AdjustMoles(gas,
-                    MathF.Max(1f, MathF.Round(amount * MathF.Round(component.Seed.Potency) / exudeCount)));
+                    MathF.Max(0.1f, MathF.Round(amount * MathF.Round(component.Seed.Potency) / exudeCount)));
```

Comment:
after some lengthy discussion on discord, I think a better solution would be to take frezon out of the possibilities for mutations instead of just omega nerfing the amount.  The other fix is fine, no stacking gas types still, but having both these combined is too much of a nerf.

Reply:
Ill look into this